### DISCUSSION TITLE
Android App > Settings: Readability of checkboxes in dark mode

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [community/community#44114] [Mobile] [answered]
### Select Topic Area

Bug

### Body

- Have Android app on dark (v1.96.0-beta at the moment)
- Go to Settings > Notifications > Working hours
- See the checkboxes

(Looks like a hardcoded color - I think they should use the accent color?)
![image](https://user-images.githubusercontent.com/65828559/212093885-4b38e7f2-8982-4a02-b6e2-dfb37417544e.png)

**Comment by alcere** [comment, marked as answer]
Thanks for reporting @Jay-o-Way! I am sharing this issue with the team, and we will start fixing it soon.

**Reply by alcere** [reply]
Hi @Jay-o-Way this got fixed and it will released this week as part of ~Beta 1.98.0~ 1.97.0.

**Reply by Jay-o-Way** [reply]
I see it already. That's nice and quick! 🙂 Wish all the bugs would be solved that fast, haha

**Comment by ghost** [comment]
> Thanks for reporting @Jay-o-Way! I am sharing this issue with the team, and we will start fixing it soon.

**Comment by perezz281** [comment]
- [ ]
